gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/setBilling.ts

```diff
@@ -37,36 +39,90 @@ export default class SetBilling extends SolanaCommand {
     this.requireFlag('state', 'Provide a valid state address')
   }
 
-  execute = async () => {
+  makeRawTransaction = async (): Promise<SolanaRawTransaction> => {
```

Comment:
This probably should return a set of txs. There could be cases where many tx are needed to perform the operation